From 44ec6e9db4494d46a2b3c4b8fada20e1e4ddc055 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 5 Sep 2011 09:53:56 +0800
Subject: ENGR00155995 [MX6]Adjust default thermal point

Current thermal reading formula is not accurate,
and different board has different value, previous
setting of trip point setting is too low, and some
boards can reach hot and critical point easily, so
change the trip point as below:

critical : 50 -> 100 C
hot : 40 -> 90 C
active : 30 -> 80 C

these trip points value can also be changed via echo
an value into /sys/class/thermal/thermal_zone0/trip..

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 drivers/mxc/thermal/thermal.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mxc/thermal/thermal.c b/drivers/mxc/thermal/thermal.c
index 84c7a63..6ec7406 100644
--- a/drivers/mxc/thermal/thermal.c
+++ b/drivers/mxc/thermal/thermal.c
@@ -114,9 +114,9 @@
 #define _COMPONENT		ANATOP_THERMAL_COMPONENT
 #define KELVIN_OFFSET			273
 #define POLLING_FREQ			2000 /* 2s */
-#define TEMP_CRITICAL			323 /* 50 C*/
-#define TEMP_HOT				313 /* 40 C*/
-#define TEMP_ACTIVE				303 /* 30 C*/
+#define TEMP_CRITICAL			373 /* 100 C*/
+#define TEMP_HOT				363 /* 90 C*/
+#define TEMP_ACTIVE				353 /* 80 C*/
 #define MEASURE_FREQ			327  /* 327 RTC clocks delay, 10ms */
 #define CONVER_CONST			14113  /* need calibration */
 #define CONVER_DIV				17259
-- 
1.7.9.5

